docs/proposals/llm-d.md

```diff
@@ -130,14 +138,28 @@ llm-d intends to use [NIXL](https://github.com/ai-dynamo/nixl) to optimize GPU o
 
 NVIDIA Dynamo offers an excellent integrated stack for low-latency and high scale serving. llm-d intends to work closely with the Dynamo team on intergrating components of Dynamo into the operational framework of Kubernetes. We are prioritizing the inference scheduler as the key component to enhance Dynamo.
 
+Unlike Dynamo, llm-d:
+* Prefers to make the prefill/decode selection within the scheduler and to trade off between latency and throughput dynamically
+* Wants to have a strong separation between in-memory prefix cache tiers and storage tiers
+
 ### Use AIBrix
 
 AIBrix provides a strong research-focused and fast iterating integrated serving platform. llm-d intends to work closely with the AIBrix team to leverage their experience in autoscaling and serving to standardize components and best practices.
 
+Unlike AIBrix, llm-d:
+* Prefers to adapt serving to existing user infrastructure vs providing an opinionated serving platform
+* Prioritizes standardizing Kubernetes upstream APIs in the Gateway ecosystem to drive alignment across many deployments
+
 ### Use Production Stack
 
 production-stack is the easiest way to deploy vLLM on Kubernetes. llm-d intends to work closely with the production-stack team to find common components and patterns to integrate, especially around prefix cache configuration.
 
+Unlike production-stack, llm-d:
+* Is focused on the needs of large scale production serving and expects users to have significant opinions about the rest of the infrastructure.
+
 ### Use KServe
 
 KServe assists platform teams in running large numbers of traditional and generative models on Kubernetes densely. Consider KServe if you have lots of LLM deployments smaller than several hosts or if you have many teams that need distinct deployments of models. llm-d focuses on operationalizing large models in very large deployments, as well as having multiple teams using a single shared deployment efficiently.
+
+Unlike KServe, llm-d:
+* Is focused on the needs of large scale production serving and expects users to desire to consume a limited set of dependencies required for that workload.
```

Comment:
I would drop the first part and just say:

- "Expects expects users to desire to consume a limited set of dependencies required for their workloads"